README.md

```diff
@@ -110,7 +103,48 @@ User.create(email: 'xxxxxx', password: 'xxxxxx')
+
+### Cleanup local Database
+```
+brew services restart postgres
+bundle exec rake db:drop db:create db:schema:load db:seed
+```
+
+## Commands
+- `bundle exec rake lint` - Run the full suite of linters on the codebase.
+- `bundle exec guard` - Runs the guard test server that reruns your tests after files are saved. Useful for TDD!
+- `bundle exec rake security` - Run the suite of security scanners on the codebase.
+- `bundle exec rake ci` - Runs the continuous integration scripts which includes linters, security scanners, tests, and code coverage
+- `bundle exec rspec spec/path/to/spec` - Run a specific spec
+
+
+## Fetching Data from the College Scorecard API
+The gibct-data-service utilizes the U.S. Department of Education's College Scorecard API to retrieve some of the data for institutions that are displayed in the Comparison Tool. After obtaining and configuring your API key as described in the "Environment Variables" section of this README above, it is relatively trivial to fetch the latest data from the API.
+
+With the GIBCT Data Service Running locally, log in to access the dashboard. From there click the green "Fetch" button on the `Scorecard` row. The Institution data will be fetched automatically from the College Scorecard API and should return a success message when complete.
+
+Institutions have a one to many relationship with their associated degree programs. To fetch the latest data for the institution degree programs, click the green "Fetch" button on the `ScorecardDegreeProgram` row. The latest ScoreCardDegree program data will be fetched and you should receive a success message when complete.
+
+## Version Generation
+Much of the data in the gibct-data-service is used to build instances of institutions to display relevant data to users of the comparison tool for particular institutions. Since the data comes in as various CSV types to build these institution objects, a versioning system is necessary to ensure the correct data is being used when building the institution objects and only approved information is released to production. As mentioned in the "Data Modes and Versions" section above, there are versioned preview and production modes of the institutions that are built from the data in the uploaded CSVs. 
+
+To generate a new preview version you must first upload any CSVs that contain changes that you wish to see in the new version of institutions being built. After you are satisfied with what has been uploaded, you must generate a new preview version by clicking "Generate New Preview Version" under the "Latest Preview Version" header on the GIBCT Dashboard. This will increment the preview version and build a new preview data set by running active record queries on the various data using `gibct-data-service/app/models/institution_builder.rb` and produce the new institution objects. You will receive a success message when this is complete.
```

Comment:
`gibct-data-service/app/models/institution_builder.rb` could be a link to the file